homeassistant/components/mobile_app/webhook.py

```diff
@@ -538,3 +540,14 @@ async def webhook_get_config(hass, config_entry, data):
         pass
 
     return webhook_response(resp, registration=config_entry.data)
+
+
+@WEBHOOK_COMMANDS.register("get_yaml_config")
```

Comment:
This name is weird, it should describe what we return, not the format of the data how it was supplied

Reply:
Agreed! `get_mobile_app_config`?

Reply:
Changed